fuel-core/src/state/rocks_db.rs

```diff
@@ -229,7 +230,13 @@ impl BatchOperations for RocksDb {
                 }
             }
         }
-
+        #[cfg(feature = "prometheus")]
+        {
+            DATABASE_METRICS.write_meter.inc();
+            DATABASE_METRICS
+                .bytes_written_meter
+                .inc_by(batch.len() as u64);
```

Comment:
Looking at the [docs](https://docs.rs/rocksdb/0.17.0/rocksdb/struct.WriteBatch.html#method.len), it seems like we should be using `size_in_bytes` instead:

```suggestion
                .inc_by(batch.size_in_bytes() as u64);
```

Reply:
good catch